app/views/repositories/_repository.html.erb

```diff
@@ -1,8 +1,16 @@
 <%= render partial: "repositories/delete_record_modal.html.erb" %>
 <%= render partial: "repositories/delete_column_modal.html.erb" %>
+<%= render partial: 'repositories/export_repository_modal.html.erb',
+           locals: { repository: repository } %>
 
 <div id="alert-container"></div>
 
+<% if can_view_repository(repository) %>
+  <%= bootstrap_form_tag(url: export_repository_team_path(repository),
+                        html: { id: 'form-export', class: 'hidden' }) do |f| %>
+  <% end %>
+<% end %>
```

Comment:
No need for separate hidden form to hold the export form data - you can put this form code inside of **a _export_repository_modal.html.erb**.